lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -736,7 +740,7 @@ def test_dates_tab_link_render(self, url_name, mfe_active):
 
         def assert_html_elements(assert_function, user):
             self.client.login(username=user.username, password=TEST_PASSWORD)
-            with override_experiment_waffle_flag(COURSE_HOME_MICROFRONTEND, active=mfe_active):
+            with override_waffle_flag(COURSE_HOME_USE_LEGACY_FRONTEND, active=(not mfe_active)):
```

Comment:
Nit: To make this more clear, I'd maybe just flip the variable name from `mfe_active` to `legacy_view_active` and then you can stay in truthy land.